### PR TITLE
Library API and its tests

### PR DESCRIPTION
**Pull request by AdrienaC**
Created for issue  #70 . Allows document search by text query or bookmark. A text query will
include 10 results from the current page, and data about the distribution of search results with regard to fields, content type, language, and so on. It will also include the number of search results, pages, and the current page, to allow for search by page.

### CODE REVIEW

File: PittAPI/library.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    def __init__(self):
+        super().__init__()
+        self.reset()
+        self.data = []
+    def handle_data(self, d):
+        self.data.append(d)
+    def get_data(self):
+        return ''.join(self.data)
+
+sess = requests.session()
```
Comment (RitwikGupta) [inline PittAPI/library.py]: I'd move this above by `LIBRARY_URL` since it's a global var

**Comment by dzheng256** [comment]
Yeah, looks good. Is this WIP finished then? @AdrienaC

**Comment by AdrienaC** [comment]
It's good for now, though I'll add more advanced search features at some point. @dzheng256

**Comment by dzheng256** [comment]
Yep, looks good.

**Comment by RitwikGupta** [comment]
Do you want to merge it now or leave it as a WIP? If you want to merge now, just rename it and comment that you've renamed it and I'll merge! @AdrienaC

**Comment by AdrienaC** [comment]
I've changed the name of the PR. It's good to go now! @RitwikGupta

**Pull request merged**